base/linalg/tridiag.jl

```diff
@@ -314,6 +322,14 @@ immutable Tridiagonal{T} <: AbstractMatrix{T}
     du2::Vector{T}   # supsup-diagonal for pivoting
 end
 
+"""
+    Tridiagonal(dl, d, du)
+
+Construct a tridiagonal matrix from the lower diagonal, diagonal, and upper diagonal,
```

Comment:
I think these should be super and sub instead of upper and lower. I think it's more common and it's how they are named in the `SymTridiagonal` docstring.